homeassistant/helpers/device.py

```diff
@@ -26,7 +26,11 @@ def async_device_info_to_link_from_entity(
     hass: HomeAssistant,
     entity_id_or_uuid: str,
 ) -> dr.DeviceInfo | None:
-    """DeviceInfo with information to link a device to a configuration entry in the link category from a entity id or entity uuid."""
+    """DeviceInfo with information to link a device from an entity.
+
+    DeviceInfo will only return the information to categorize the config
```

Comment:
We shouldn't talk about config entry here, I think. We're returning a device entry and it's that item that belongs to a link category here.